fourkitchens/suhumsci
# SHS-4985: Make ckeditor media elements non clickable

# READY FOR REVIEW

## Summary
- Make ckeditor media elements non clickable

## Steps to Test
1. Create a flexible page. Add a video in the Main Region text area.
2. Confirm that you can click anywhere within the video to access the contextual toolbar without playing or interacting with the video in any way.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
